cpp/test/utils/metrics/log_metric_test.cc

```diff
@@ -0,0 +1,150 @@
+#include "src/utils/metrics/log_metric.hh"
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <filesystem>
+#include <fstream>
+#include <regex>
+#include <string>
+#include <vector>
+
+#include "src/utils/logging.hh"
+
+namespace torchserve {
+class TSLogMetricTest : public ::testing::Test {
+ protected:
+  const std::string logfile_path{"test/resources/logging/test.log"};
+  const std::string logger_config_path_str{
+      "test/resources/logging/log_to_file.config"};
```

Comment:
where is the log_to_file.config?

Reply:
It is here: https://github.com/pytorch/serve/blob/cpp_backend/cpp/test/resources/logging/log_to_file.config
and is used for the `logging` unit tests. I've re-used the same here.